runtime/common/src/migrations.rs

```diff
@@ -595,7 +621,10 @@ pub struct XcmMigrations<Runtime, StatemineParaIdInfo, StatemineAssetsInstanceIn
 impl<Runtime, StatemineParaIdInfo, StatemineAssetsInstanceInfo> GetMigrations
 	for XcmMigrations<Runtime, StatemineParaIdInfo, StatemineAssetsInstanceInfo>
```

Comment:
the migration was configured in XcmMigrations

please move it into `CommonMigrations` impl of `GetMigrations` further above and can revert adding the `parachain_staking::Config` trait bound to `XcmMigrations` impl

Reply:
good catch